catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorListFragment.java

```diff
@@ -135,6 +136,23 @@
 	private String actionBarTitle;
 	private int[] itemsIds;
 
+	public FormulaEditorListFragment() {
```

Comment:
the default constructor would be autogenerated by the compiler -> can be removed